src/main/java/com/couchbase/lite/router/Router.java

```diff
@@ -1563,6 +1564,7 @@ public Status do_GET_Document_changes(Database _db, String docID, String _attach
             Log.v(Log.TAG_ROUTER, "Filter params=" + changesFilterParams);
         }
 
+        // changesSince() is query only. not required synchronized
         RevisionList changes = db.changesSince(since, options, changesFilter, changesFilterParams);
```

Comment:
As [changesSince()](https://github.com/couchbase/couchbase-lite-java-core/blob/a35361bc47ecb6c55849694deece22fa4aaf7c62/src/main/java/com/couchbase/lite/store/SQLiteStore.java#L963) does not access shard instance and SQLiteDatabae is thread-safe as default, I believe this method is thread-safe.